arm/Microsoft.Network/virtualNetworkGateways/deploy.bicep

```diff
@@ -74,7 +74,7 @@ param clientRevokedCertThumbprint string = ''
 @maxValue(365)
 param diagnosticLogsRetentionInDays int = 365
 
-@description('Required. Resource identifier of the Diagnostic Storage Account.')
+@description('Required. Resource ID of the Diagnostic Storage Account.')
```

Comment:
Not related to this PR but since I'm noticing: `Optional` here

Reply:
Fixed

Reply:
I still see `Required`

Reply:
I don't. But the workspace had it too. Fixed it there